### ISSUE TITLE
TypeError (strcasecmp) on package scan

### ISSUE THREAD

**Issue by mbrodala** [composer/packagist#1318]
Yesterday I pushed version [0.10.0 of t3g/file-variants](https://github.com/TYPO3GmbH/ext-file-variants/releases/tag/0.10.0).

However, this version does not show up on https://packagist.org/packages/t3g/file-variants. A manual update yields the following error:

![image](https://user-images.githubusercontent.com/5037116/175480620-ca4a1582-a3db-4dca-a98a-3d454950e271.png)

> An unexpected failure occurred [TypeError]: strcasecmp(): Argument #1 ($string1) must be of type string, int given

Currently I cannot see any error in the [Composer manifest](https://github.com/TYPO3GmbH/ext-file-variants/blob/0.10.0/composer.json), only the license gets a warning:

```
$ composer validate
./composer.json is valid, but with a few warnings
See https://getcomposer.org/doc/04-schema.md for details on the schema
# General warnings
- License "GPL-2.0+" is a deprecated SPDX license identifier, use "GPL-2.0-or-later" instead
```

Any idea what's wrong?

**Comment by mbrodala** [comment]
Good catch, we did indeed fix that with 0.9.1:

https://github.com/TYPO3GmbH/ext-file-variants/commit/e56386262867c62a4cc243b10de0b398e809c9ef

Thanks for having a look at this!

**Comment by mbrodala** [comment]
Thanks for the better error message. What can we do about the issue at hand? We still cannot see the new tag on Packagist ...

**Comment by Seldaek** [comment]
@mbrodala ah sorry I figured if you released a fix in 0.9.1 it wasn't so urgent to fix this in prod. But now I see this is <0.10 so I guess you're still waiting :)

I'll get this deployed soon.

**Comment by mbrodala** [comment]
Thanks for the clarification, both of you!
